### DISCUSSION TITLE
R-link error no data pyren

### DISCUSSION THREAD

**Discussion by Alecsbdj** [cedricp/ddt4all#1167] [Q&A]
Howdy! I have a
the problem is, after I implement the macro from VIN, I get these errors. do you have any idea
Error_question = 1
Error_nodata = 28
![Uploading IMG_2626.jpeg…]()

**Comment by AndyIreland** [comment]
pic didnt show up

**Reply by Alecsbdj** [reply]
![IMG_2806](https://github.com/user-attachments/assets/b45273b8-c747-493f-a817-f6dbf6e32e76)

**Comment by Furtif** [comment]
Is ddt4all git not pyren !

**Reply by AndyIreland** [reply]
PYREN= Gitlab = https://gitlab.com/py_ren/pyren/-/issues
DDT4ALL = Github = https://github.com/cedricp/ddt4all/discussions

hope this helps
